handsontable/src/core/hooks/constants.js

```diff
@@ -1886,6 +1886,16 @@ export const REGISTERED_HOOKS = [
    */
   'modifyFocusOnTabNavigation',
 
+  /**
+   * Used to modify the cell coordinates when the table is deactivated (going out of the listen mode).
+   *
+   * @event Hooks#modifyUnfocusOnTabNavigation
+   * @since 16.1.0
+   * @param {'to_above' | 'to_below'} tabActivationDir The browsers Tab navigation direction. Depending on
+   * whether the user deactivated the table from the element above or below, another cell can be selected.
```

Comment:
The table's default behavior after the focus table exits can be skipped. Missing the `@return` tag with description.